frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/StatusField.tsx

```diff
@@ -121,23 +128,35 @@ export function StatusField(props: { cluster: Cluster }) {
                 </AcmButton>
             )
             break
-        case ClusterStatus.creating:
+        case ClusterStatus.creating:        
         case ClusterStatus.destroying:
         case ClusterStatus.provisionfailed:
             hasAction = true
-            Action = () => (
-                <AcmButton
-                    style={{ padding: 0, fontSize: 'inherit' }}
-                    key={props.cluster.name}
-                    onClick={() => launchLogs(props.cluster, configMaps)}
-                    variant="link"
-                    role="link"
-                    icon={<ExternalLinkAltIcon />}
-                    iconPosition="right"
-                >
-                    {t('view.logs')}
-                </AcmButton>
-            )
+            if ( isHybrid ) { //Correct link to installation logs in assisted clusters
+                Action = () => (
+                    <LogsDownloadButton
```

Comment:
It would be good to align the style of the button:
```suggestion
                    <LogsDownloadButton
                      Component={(props) => (
                            <Button {...props} icon={<DownloadIcon />} iconPosition="right" isInline />
                        )}
```

Reply:
<img width="366" alt="Screenshot 2022-03-14 at 10 05 19" src="https://user-images.githubusercontent.com/1121740/158139935-3a987451-f72a-4bda-92d8-8d520fef7ae9.png">